www/src/public/js/controllers/builder/main.js

```diff
@@ -366,20 +376,23 @@
 
                 itemIndex++;
             }
-            
+
             // v3: Adds missing "other" slots to previous lists
-            if(itemIndex < $scope.slotOrder.length) {
+            if (itemIndex < $scope.slotOrder.length && listVersion < $scope.listVer) {
                 while (itemIndex < $scope.slotOrder.length){
                     itemIndex++
                     items.push({"id": Number([0]), "slot": 21});
                 }
             }
-
-            // Version Check, if # of item slots loaded does not equal total number in current version it will throw an error
-            if(itemIndex > $scope.slotOrder.length) {
-                throw "Invalid list.";
+            
+            // Version Check, if # of item slots loaded does not equal total number in current version it will throw an error and attempt to parse the list with a different version in the try/catch
+            if(itemIndex != $scope.slotOrder.length) {
```

Comment:
good change, but I wonder, do we really need this check anymore? We were only using it to differentiate version 2 and 3, but now version 3 is in the `cln` var, so we shouldn't need this to differentiate between v1 and v2. Maybe double check that.